host/tests/L3_TestCases/dsVideoPort/dsVideoPort_test10_VerifyALLM_Mode.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+#** *****************************************************************************
+# *
+# * If not stated otherwise in this file or this component's LICENSE file the
+# * following copyright and licenses apply:
+# *
+# * Copyright 2024 RDK Management
+# *
+# * Licensed under the Apache License, Version 2.0 (the "License");
+# * you may not use this file except in compliance with the License.
+# * You may obtain a copy of the License at
+# *
+# *
+# http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# *
+#* ******************************************************************************
+
+import os
+import sys
+
+# Get directory path and append to system path
+dir_path = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(os.path.join(dir_path, "../../"))
+
+# Import required classes from modules
+from dsVideoPortHelperClass import dsVideoPortHelperClass
+from raft.framework.core.logModule import logModule
+
+class dsVideoPort_test10_VerifyALLM_Mode(dsVideoPortHelperClass):
+    """
+    A class to test and verify Enables/Disables ALLM mode for HDMI output video port.
+
+    Attributes:
+        testName (str): Name of the test.
+        testSetupPath (str): Path to the test setup configuration file.
+        moduleName (str): Name of the module being tested.
+        rackDevice (str): Device under test (DUT).
+    """
+
+    def __init__(self, log:logModule=None):
+        """
+        Initializes the test10_VerifyALLM_Mode test setup and configuration.
+
+        Initializes sessions, reads the test setup, and prepares the user response.
+        """
+        self.testName  = "test10_VerifyALLM_Mode"
+        self.qcID      = '10'
+
+        super().__init__(self.testName, self.qcID, log)
+
+
+    #TODO: Current version supports only manual verification.
+    def testVerifyAllmMode(self, manual=False):
+        """
+        Verifies Enables/Disables ALLM mode for HDMI output video port..
+
+        Args:
+            manual (bool, optional): If True, manual verification is done using user response; otherwise,
+                                     automated verification is used (yet to be implemented).
+
+        Returns:
+            bool: Result of the ALLM mode verification.
+        """
+        if manual == True:
+            #hdcpVersion = self.testdsVideoPort.getHDCPVersion()
```

Comment:
can you please remove the commented line if it is not required.

Reply:
Removed the commented line.